packages/google_sign_in/google_sign_in_ios/darwin/Tests/GoogleSignInTests.m

```diff
@@ -55,10 +61,10 @@ - (void)testDisconnect {
 }
 
 - (void)testDisconnectIgnoresError {
-  NSError *error = [NSError errorWithDomain:kGIDSignInErrorDomain
-                                       code:kGIDSignInErrorCodeHasNoAuthInKeychain
-                                   userInfo:nil];
-  [[self.mockSignIn stub] disconnectWithCallback:[OCMArg invokeBlockWithArgs:error, nil]];
+  NSError *sdkError = [NSError errorWithDomain:kGIDSignInErrorDomain
```

Comment:
All the error renaming is because the macOS build settings catch variable shadowing, which apparently our iOS build doesn't have enabled, and the blocks in all of these methods have an `error` parameter.

Reply:
Is there an issue filed to get the iOS lints consistent with the macOS lints? (also raises the question of whether macOS is missing any iOS lints)